http_server_spec/features/01_getting_started/simple_head.feature

```diff
@@ -1,13 +1,11 @@
 @simple-head @01-getting-started
 Feature: Simple HEAD
 
-  @wip
   Scenario: Executing a HEAD request to /simple_get
     Given I make a HEAD request to "/simple_get"
     Then my response should have status code 200
     And my response should have an empty body
 
-  @wip
   Scenario: HEAD does not include body
     Given I make a HEAD request to "/head_request"
     Then my response should have status code 200
```

Comment:
I think it would be super cool if you added to this acceptence test.

Scenario: GET request does include the body
    Given I make a GET request to "/head_request"
    Then my response should have status code 200
    And my response should have a body with the text "This body does not show up in a HEAD request"

Reply:
you know some Ruby, and it would be a nice challenge to see if you could figure out how to implement this.

Reply:
also, if it works, you could make a PR against the apprentice repo, so future apprentices will have the extra help as well.

Reply:
How does [this](https://github.com/t-keazirian/HTTPServer/pull/24/commits/85850d4989b812c26f2c9c8c2d339508aa46b5f6) look?

Reply:
that looks great! how bout that fast turn around!

Reply:
![Uploading image.png…]()
:tada: